draft-packages/menu-list/KaizenDraft/MenuList/MenuList.scss

```diff
@@ -31,13 +31,13 @@ $side-padding: 3/4 * $ca-grid;
 }
 
 .header__title {
-  @include ca-type-ideal-labels-and-legends;
+  @include ca-type-inter-labels-and-legends;
```

Comment:
```suggestion
  @include kz-typography-heading-6;
```

Reply:
Not sure if this mixin is available in `component-library/styles/type`, so might need to add `@import "~@kaizen/deprecated-component-library-helpers/type";`